src/hooks/use_profile.ts

```diff
@@ -26,6 +26,16 @@ const fetchLink = async (address: string) => {
   return data.data;
 };
 
+const fetchDesmosDtag = async (dtag: string) => {
+  const { data } = await axios.post(PROFILE_API, {
+    variables: {
+      dtag,
+    },
+    query: DesmosProfileDocumentDtag,
+  });
+  return data.data;
+};
+
```

Comment:
`fetchDtag` sounds more efficient..maybe..
fetchDesmos == i'm fetching a desmos native address
fetchLink == i'm fetching an address that's not desmos but has a link connection to a desmos address
fetchDtag == i'm fetching a dtag

Reply:
This makes sense.